pkg/quay/quay.go

```diff
@@ -5,27 +5,31 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"os"
 	"time"
 
 	"github.com/giantswarm/microerror"
 
+	"github.com/giantswarm/crsync/internal/env"
 	"github.com/giantswarm/crsync/pkg/registry"
 )
 
 const (
-	publicImagesOnly   = true
 	registryEndpoint   = "https://quay.io"
 	repositoryEndpoint = "https://quay.io/api/v1/repository"
 )
 
 type Config struct {
-	Namespace    string
-	LastModified time.Duration
+	Namespace               string
+	LastModified            time.Duration
+	SyncPrivateRepositories bool
```

Comment:
This object doesn't sync per-se. I found this setting confusing initially.

```suggestion
	IncludePrivateRepositories bool
```